eng/testing/linker/trimmingTests.targets

```diff
@@ -68,6 +69,16 @@
       <_additionalProjectSourceFiles Include="%(TestConsoleApps.AdditionalSourceFiles)" />
     </ItemGroup>
 
+    <ItemGroup>
+      <_switchesAsItems Include="%(TestConsoleApps.DisabledFeatureSwitches)" Value="false" />
+      <_switchesAsItems Include="%(TestConsoleApps.EnabledFeatureSwitches)" Value="true" />
+    </ItemGroup>
+
+    <PropertyGroup>
+      <_extraTrimmerArgs>$(_extraTrimmerArgs) @(_switchesAsItems->'--feature %(Identity) %(Value)', ' ')</_extraTrimmerArgs>
+      <_runtimeHostConfigruationOptionsString>@(_switchesAsItems->'&lt;RuntimeHostConfigurationOption Include=&quot;%(Identity)&quot; Value=&quot;%(Value)&quot; /&gt;', '%0a    ')</_runtimeHostConfigruationOptionsString>
```

Comment:
```suggestion
      <_runtimeHostConfigurationOptionsString>@(_switchesAsItems->'&lt;RuntimeHostConfigurationOption Include=&quot;%(Identity)&quot; Value=&quot;%(Value)&quot; /&gt;', '%0a    ')</_runtimeHostConfigurationOptionsString>
```

Reply:
🤦

Reply:
Actually, do we really need this extra layer of infrastructure? Why not just continue with the test setting `--feature` in the ExtraTrimmerArgs? That has been working so far.

Reply:
The feature switch for built-in COM reaches across libraries and runtime, so it needs to actually be set in the runtime config properties instead of just relying on the linker to substitute on the libraries side.

(It could also make sense to make the other ones use this since I think it is closer to the SDK experience? Unless we are intentionally checking that just passing the linker args is sufficient)

Reply:
>  it needs to actually be set in the runtime config properties instead of just relying on the linker to substitute on the libraries side.

Ah, that makes sense.

> It could also make sense to make the other ones use this since I think it is closer to the SDK experience?

If you wanted to convert those other ones as well, I think that would be great.

Reply:
I will do a follow-up to convert the others once we get this in.